README.md

```diff
@@ -133,7 +133,85 @@ Be careful; while you can inspect and change even more as root, Bottlerocket's f
 Rather than a package manager that updates individual pieces of software, Bottlerocket downloads a full filesystem image and reboots into it.
 It can automatically roll back if boot failures occur, and workload failures can trigger manual rollbacks.
 
-Currently, you can update using a CLI tool, updog.
+The update process uses images secured by [TUF](https://theupdateframework.github.io/).
+For more details, see the [update system documentation](sources/updater/).
+
+There are several ways of updating your Bottlerocket instances:
+
+#### Update API
+
+The [Bottlerocket API](#api) allows you to update the instance from within the [control container](#control-container) or any [host container](#custom-host-containers) that has access to the Bottlerocket API.
+
+The update API is made up of a set of actions endpoints and a status endpoint to check the result of those actions.
+The behavior of the update API is governed by the [update settings](#updates-settings).
+
+In general, the update process looks like the following:
+
+Refresh list of available updates:
+```
+apiclient -u /actions/refresh-updates -m POST
+```
+
+Then check the list of available updates with:
+```
+apiclient -u /updates/status
+```
+
+This will return the current update status in JSON format. For example, the status should look something like the following (pretty-printed):
+```
+{
+  "update_state": "Available",
+  "available_updates": [
+    "0.4.0",
+    "0.3.4",
+    ...
+  ],
+  "chosen_update": {
+    "arch": "x86_64",
+    "version": "0.4.0",
+    "variant": "aws-k8s-1.15"
+  },
+  "active_partition": {
+    "image": {
+      "arch": "x86_64",
+      "version": "0.3.2",
+      "variant": "aws-k8s-1.15"
+    },
+    "next_to_boot": true
+  },
+  "staging_partition": null,
+  "most_recent_command": {
+    "cmd_type": "refresh",
+    "cmd_status": "Success",
+    ...
+  }
+}
+```
+
+If the `chosen_update` (see `settings.updates.version-lock` in [update settings](#updates-settings)) looks right, you can prepare said update with:
+```
+apiclient -u /actions/prepare-update -m POST
+```
+
+After update gets being downloaded and applied, you can check the update status again with:
+```
+apiclient -u /updates/status
+```
+
+If the `staging_partition` image information looks right (should be the chosen update), you can proceed to "activate" the update with:
```

Comment:
```suggestion
If the staging partition shows the new version, you can proceed to "activate" the update.  This means that as soon as the host is rebooted it will try to run the new version.  (If the new version can't boot, we automatically flip back to the old version.)
```